### ISSUE TITLE
ARIA role should be appropriate for the element

### ISSUE THREAD

**Issue by fmazeiras** [AdeleD/react-paginate#492]
since version 8.1.4 the component have accessibility violation when using axe to check

the error is

<ul class="paginateContainerClassName" role="navigation" aria-label="Pagination">
    Received:
    "ARIA role should be appropriate for the element (aria-allowed-role)"
    Fix any of the following:
      ARIA role navigation is not allowed for given element
    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/4.5/aria-allowed-role?application=axeAPI

the role navigation is not allow for ul elment

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role

**Comment by christoshrousis** [comment]
Does not pass [jest-axe](https://github.com/nickcolley/jest-axe)

![Screenshot 2023-08-25 at 1 27 15 pm](https://github.com/AdeleD/react-paginate/assets/951957/0a8f0dfa-f541-465b-abff-2e24e88f4fe6)

https://dequeuniversity.com/rules/axe/4.4/aria-allowed-role?application=axeAPI

**Comment by martinholecekmax** [comment]
Same problem here. Any idea how to fix it?

**Comment by AdrianFahrbach** [comment]
Same here! Would be nice to get this fixed, since this should be a one-liner.

**Comment by MonikaPatelIT** [comment]
Same here. Let us pass role as prop if it's okay

**Comment by fandporsh** [comment]
Hello same problem don't pass Jest-Axe 

in file PaginationBoxView.js

```
      <ul
        className={className || containerClassName}
        role="navigation"
        aria-label="Pagination"
      >
```
no need role for <ul>

**Comment by fandporsh** [comment]
> Hello same problem don't pass Jest-Axe
> 
> in file PaginationBoxView.js
> 
> ```
>       <ul
>         className={className || containerClassName}
>         role="navigation"
>         aria-label="Pagination"
>       >
> ```
> 
> no need role for ul

**Comment by ViktorSoroka07** [comment]
+1

**Comment by jessie-james-jg** [comment]
This  is actually a bug in v8.1.3.    v8.1.2  is the last version that does not have this accessibility violation.

**Comment by fmazeiras** [comment]
Hello,

i receive this comment "This is actually a bug in v8.1.3. v8.1.2 is the last version that does not have this accessibility violation." by email but it does not appear here.

I can confirm that this issue is still in the last version 8.1.5 because the pull request that can fix it is not merged  fix #459: Remove incorrect aria role navigation #494 

regards
